.github/actions/setup-kurtosis-type1/action.yml

```diff
@@ -15,17 +15,45 @@ runs:
       uses: actions/checkout@v4
       with:
         repository: 0xPolygon/kurtosis-cdk
-        ref: v0.4.4
+        ref: v0.4.6
         path: kurtosis-cdk
 
     - name: Install Kurtosis CDK tools
       if: env.SKIP_ENCLAVE_SETUP != 'true'
-      uses: ./kurtosis-cdk/.github/actions/setup-kurtosis-cdk
+      uses: ./kurtosis-cdk/.github/actions/kurtosis-pre-run
 
     - name: Install Foundry
       if: env.SKIP_ENCLAVE_SETUP != 'true'
       uses: foundry-rs/foundry-toolchain@v1
 
+    - name: Install polycli
+      shell: bash
+      run: |
+        tmp_dir=$(mktemp -d) && curl -L https://github.com/0xPolygon/polygon-cli/releases/download/v0.1.48/polycli_v0.1.48_linux_amd64.tar.gz | tar -xz -C "$tmp_dir" && mv "$tmp_dir"/* /usr/local/bin/polycli && rm -rf "$tmp_dir"
+        sudo chmod +x /usr/local/bin/polycli
+        /usr/local/bin/polycli version
+
+    - name: Install yq
```

Comment:
yq is installed by kurtosis-pre-run action I believe

Reply:
It is indeed, but I had problems with the configs not being manipulated, and these were resolved by bringing this step back in.